lightning-invoice/src/utils.rs

```diff
@@ -995,9 +993,9 @@ mod test {
 	#[cfg(feature = "std")]
 	fn do_test_multi_node_receive(user_generated_pmt_hash: bool) {
 		let mut chanmon_cfgs = create_chanmon_cfgs(3);
-		let seed_1 = [42 as u8; 32];
-		let seed_2 = [43 as u8; 32];
-		let cross_node_seed = [44 as u8; 32];
+		let seed_1 = [42_u8; 32];
+		let seed_2 = [43_u8; 32];
+		let cross_node_seed = [44_u8; 32];
```

Comment:
Hmm, but this should have enforced no underscores: https://github.com/lightningdevkit/rust-lightning/pull/2049#discussion_r1118827411

Reply:
Weird, it seems `-Dclippy::separated_literal_suffix` only works if the code is already using underscores. So this is a instance where you need to run clippy twice to get the final result

Reply:
So I run clippy again and now it's correct

Reply:
> Weird, it seems `-Dclippy::separated_literal_suffix` only works if the code is already using underscores. So this is a instance where you need to run clippy twice to get the final result

Indeed for a small test project it works running only once, but maybe `rust-lightning` is too big and some heuristics kick in to avoid this extra computation.